packages/salesforcedx-sobjects-faux-generator/package.json

```diff
@@ -34,6 +34,7 @@
   },
   "scripts": {
     "compile": "webpack",
+    "webpack": "webpack",
```

Comment:
interesting..did something have an issue with this being the compile task?

Reply:
I'm guessing to make it consistent across packages.

Reply:
Yes, this is so `lerna run webpack` can be invoked from post compile step instead of using `lerna exec --scope ` with regex from the root package to select the right packages which was a bit error prone. Also find the consistency of leaving the packages to deal with it a bit cleaner. Same applies for the post compile step to copy files.